demo/vl-search-results.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+	<head>
+        <title>vl-search-results voorbeeld</title>
+        <script type="module" src="/node_modules/vl-ui-demo/dist/vl-demo-all.js"></script>
+
+        <script type="module" src="/src/vl-search-results.js"></script>
+        <script type="module" src="/src/vl-search-result.js"></script>
+
+        <link rel="stylesheet" type="text/css" href="/src/style.css" />
+    </head>
+
+	<body>
+        <vl-demo-page data-vl-webcomponent="vl-search-results" data-vl-link="https://overheid.vlaanderen.be/webuniversum/v3/documentation/components/vl-ui-search-results">
+            <vl-demo data-vl-title="Search results">
+                <vl-search-results>
+                    <vl-search-result>
```

Comment:
Zoals in de chat ook besproken: er was momenteel (nog) geen specifieke plaats voorzien voor de actieknoppen/linken. Momenteel is het mogelijk om deze binnen <dl slot="content /> te plaatsen, maar niet er buiten want dan werden ze niet gerenderd.

Ik had aangegeven dat het voorlopig niet nodig was om dit al te voorzien. Zeker aangezien Tom ook aangaf of het niet beter is om het 'admin' gedeelte buiten deze UI te houden. Misschien moet dit nog teruggekoppeld worden met de business eerst? Of wat denkt het Phoenix Team?

Reply:
Een actions slot kan voorzien worden, maar ik zou inderdaad eerst valideren a.d.h.v. mockup dat het combineren van de gebruiker en beheerder view de beste oplossing is. Het zou kunnen dat de beheerders anders willen kunnen zoeken dan eindgebruikers en dan ben je genoodzaakt views en logica te mengen waardoor het eindresultaat potentieel voor geen van de partijen ideaal is.